### PR TITLE
Adjust for upcoming toolkit v1.0.2 that fixes SSR issue

### PR DESCRIPTION
**Pull request by RJFelix**
Hey, Ryan @ Userfront here. Thanks for building this example of how to use `@userfront/toolkit` with Svelte, and for all your feedback!

We'll publish a `@userfront/toolkit 1.0.2` in the next couple days that checks to make sure we're in the browser before trying to register the Web Components, so importing `@userfront/toolkit/web-components` on the server won't crash.

Key changes:

- Import the `Userfront` CoreJS library from `@userfront/toolkit/web-components`
  - Under the hood the components use the same CoreJS instance, so `Userfront.init(tenantId)` applies to them too.
- Now the Svelte component wrappers don't do much, so I removed them
- Added the toolkit logout button to the top nav. (I'm not sure what's going on with the button's size & vertical alignment there. I've made a note to look into it further.)
- Added a `/signup` page to demo redirecting signed-in users when the page loads. The recent toolkit revamp adds a `redirect-on-load-if-logged-in` attribute that controls this behavior. It defaults off.
  - We realized that redirect behavior can be different on-load vs. when the signup/login is completed. There's common uses for each combination of the two, so they're separate options. The default is redirect-on-load no, redirect-after-sign-in (`redirect` attribute) yes.
  - We're working on more docs on new features/differences from the legacy toolkit.

I'll update when 1.0.2 is out, and then this should work outside my local environment.

I'd love to hear any reactions or feedback you have about these changes!

### CODE REVIEW

File: package.json
Comment (Leftium) [inline package.json:1]: I think `esm-env` can be removed now, too.

File: src/routes/+layout.svelte [hunk 1]
```diff
@@ -36,9 +36,11 @@
 	<nav>
 		<ul>
 			<li><a href="/">Home</a></li>
+			<li><a href="/signup">Signup</a></li>
 			<li><a href="/login">Login</a></li>
 			<li><a href="/reset">Reset</a></li>
 			<li><a href="/dashboard">Dashboard</a></li>
+			<li style="height: 100px;"><logout-button></logout-button></li>
```
Comment (Leftium) [inline src/routes/+layout.svelte]: What's the behavior of the logout button when the user is not logged in?

Would it be worth conditionally rendering the button inside a [Svelte {#if} block](https://svelte.dev/docs/logic-blocks#if)?

**Review by Leftium — COMMENTED** [review]
Wow, thanks for making `@userfront/toolkit/web-components` SSR-friendly!

Just a couple comments inline.

File: src/routes/+layout.svelte (hunk 1 above)
Comment (RJFelix) [reply src/routes/+layout.svelte]: The Toolkit logout button displays whether or not a user is logged in, so good call on conditionally rendering. I removed it from the example to keep it simple for now. (I'm considering whether a "log in/log out" button, where the appearance and action changes based on whether the user is logged in, would make sense as a pre-built component for the Toolkit.)

**Comment by RJFelix** [comment]
Updated to `@userfront/toolkit v1.0.2`, which should eliminate the server-side error and allow the more streamlined example to work correctly.

**Pull request merged**